src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Let's try to avoid [wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements)!